docs/section19/19-10.md

```diff
@@ -0,0 +1,72 @@
+# Section 19 - Rule 19-10       
+**Schema Version:** 0.0.22  
+**Mandatory Rule:** True    
+**Rule ID:** 19-10       
+**Rule Description:** Air economizers shall be included in baseline HVAC Systems 3 through 8, and 11, 12, and 13 based on climate as specified in Section G3.1.2.6 with the following exceptions:   
+1. Systems that include gas-phase air cleaning to meet the requirements of Standard 62.1, Section 6.1.2. This exception shall be used only if the system in the  proposed design does not match the building design.
```

Comment:
I am not following the part above "...This exception shall be used only if the system in the  proposed design does not match the building design"

Reply:
90.1 committee to fix it. Jason will ask someone on ECB if they know what this is about.

Reply:
There should be two fails, one where no baseline economizer when potentially required and no proposed where exceptions could apply. The other where no economizer in baseline and yes economizer in proposed where exceptions cannot apply because there is an economizer in the proposed.

Reply:
Done

Reply:
We've discussed it with ECB and the tentative agreement is that this language should be analogous to exception 2 - economizer should be included in the baseline if it's specified in the proposed design that has gas-phase air cleanaing. We will hopefully have the final resolution for this in Atlanta.